react-form-controls/src/index.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react';
+import ReactDOM from 'react-dom/client';
+
+function RegistrationForm() {
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    // console.log('Current username state:', username);
```

Comment:
Personally, I prefer these two lines where you log from state. Either way works, though.